## ANALIZA PODATKOV

In [4]:
import pandas as pd

hribi = pd.read_csv('obdelani_podatki/hribi.csv')

In [5]:
hribi

ime     drzava  \
0                                       Snežnik  Slovenija   
1                                  Mali Snežnik  Slovenija   
2                                    Mali Golak  Slovenija   
3                                 Srednji Golak  Slovenija   
4                                       Mrzovec  Slovenija   
...                                         ...        ...   
1317                     Sveti Duh (Stara Gora)  Slovenija   
1318  Rimsko gomilno grobišče v Anželovem gozdu  Slovenija   
1319                       Babičev mlin na Muri  Slovenija   
1320                     Brod na Muri (Melinci)  Slovenija   
1321   Razkriški kot (sotočje Mure in Ščavnice)  Slovenija   

                                            gorovje  visina  \
0         Goriško, Notranjsko in Snežniško hribovje    1796   
1         Goriško, Notranjsko in Snežniško hribovje    1694   
2         Goriško, Notranjsko in Snežniško hribovje    1495   
3         Goriško, Notranjsko in Snežniško hribovje    1480   
4         Goriško, Notranjsko in Snežniško hribovje    1410   
...                                             ...     ...   
1317  Strojna, Košenjak, Kozjak in Slovenske gorice     262   
1318  Strojna, Košenjak, Kozjak in Slovenske gorice     260   
1319  Strojna, Košenjak, Kozjak in Slovenske gorice     182   
1320  Strojna, Košenjak, Kozjak in Slovenske gorice     175   
1321  Strojna, Košenjak, Kozjak in Slovenske gorice     169   

                  vrsta  stevilo_ogledov  priljubljenost  stevilo_poti  \
0             vrh, koča           851397              99            12   
1                   vrh           431337              83             4   
2                   vrh           637569              97             5   
3                   vrh           399136              79             3   
4                   vrh           414632              81             5   
...                 ...              ...             ...           ...   
1317  vrh, koča, cerkev              322              10             2   
1318                NaN              132               5             1   
1319                NaN             1453              21             2   
1320                NaN           196546              55             1   
1321                NaN               72               3             2   

                                                   opis  
0     Snežnik oz. Veliki Snežnik je z višino 1796 m ...  
1     Mali Snežnik je vrh na zahodnem grebenu Snežni...  
2     Mali Golak je najvišji od Golakov. Imena ni do...  
3     Srednji Golak se nahaja med Malim in Velikim G...  
4     Mrzovec je najizrazitejši vrh v osrčju Trnovsk...  
...                                                 ...  
1317  Stara Gora je naselje v občini Sveti Jurij ob ...  
1318  Rimsko gomilno grobišče v Anželovem gozdu v Br...  
1319  Babičev mlin na Muri se nahaja le kakih 100 me...  
1320  Brod v Melincih je eden od štirih brodov, ki š...  
1321  Razkriški kot se nahaja severno od naselja Gib...  

[1322 rows x 9 columns]

In [6]:
poti = pd.read_csv('obdelani_podatki/poti.csv')
poti

ime_poti  cas_poti  \
0                          Sviščaki - Snežnik (peš pot)       120   
1                            Kapetanova bajta - Snežnik        55   
2                              Leskova dolina - Snežnik       195   
3      Snežniška Grda jama - Snežnik (čez Mali Snežnik)        85   
4                              Gašperjev hrib - Snežnik        90   
...                                                 ...       ...   
4384                      Veržej - Babičev mlin na Muri        20   
4385                    Beltinci - Babičev mlin na Muri        95   
4386  Mota - Brod na Muri (Melinci) (od mrtvic do reke)        45   
4387  Razkrižje - Razkriški kot (sotočje Mure in Šča...        75   
4388  Gibina - Razkriški kot (sotočje Mure in Ščavnice)         6   

          tezavnost_poti                                       vrh  
0     lahka označena pot                                   Snežnik  
1     lahka označena pot                                   Snežnik  
2     lahka označena pot                                   Snežnik  
3     lahka označena pot                                   Snežnik  
4     lahka označena pot                                   Snežnik  
...                  ...                                       ...  
4384  lahka označena pot                      Babičev mlin na Muri  
4385  lahka označena pot                      Babičev mlin na Muri  
4386  lahka označena pot                    Brod na Muri (Melinci)  
4387  lahka označena pot  Razkriški kot (sotočje Mure in Ščavnice)  
4388  lahka označena pot  Razkriški kot (sotočje Mure in Ščavnice)  

[4389 rows x 4 columns]

In [10]:
st_hribov = hribi['ime'].count()
st_hribov

1322

In [12]:
st_vseh_poti = poti['ime_poti'].count()
st_vseh_poti

4389

Nekaj je šlo narobe, saj je shranjenih 2182 strani (za vsak hrib oz. vrh ena) v csv datoteki pa je zgolj 1322 vrhov. Kje se je zalomilo?
Poglejmo koliko vrhov imamo v posameznem gorovju.

In [23]:
julijci = hribi[hribi.gorovje == 'Julijske Alpe']
julijci

ime     drzava        gorovje  visina       vrsta  \
231                  Triglav  Slovenija  Julijske Alpe    2864  vrh, bivak   
232                Škrlatica  Slovenija  Julijske Alpe    2740         vrh   
233             Mali Triglav  Slovenija  Julijske Alpe    2725         vrh   
234             Visoki Rokav  Slovenija  Julijske Alpe    2646         vrh   
235                  Jalovec  Slovenija  Julijske Alpe    2645         vrh   
..                       ...        ...            ...     ...         ...   
535            Izvir Glijuna  Slovenija  Julijske Alpe     450         NaN   
536                Mala Boka  Slovenija  Julijske Alpe     440         NaN   
537               Kozlov rob  Slovenija  Julijske Alpe     426         vrh   
538  Korita Soče pri Kršovcu  Slovenija  Julijske Alpe     390         NaN   
539         Tolminska korita  Slovenija  Julijske Alpe     180         NaN   

     stevilo_ogledov  priljubljenost  stevilo_poti  \
231          1523369             100            28   
232           776333              99             5   
233           526828              92             3   
234           514309              91             2   
235           832853              99            10   
..               ...             ...           ...   
535           293607              67             1   
536           214664              57             1   
537           376188              77             2   
538           322408              70             1   
539           431701              83             0   

                                                  opis  
231  Triglav je 2864 m visoka gora, ki se nahaja v ...  
232  Škrlatica je druga najvišja gora v Sloveniji. ...  
233  Mali Triglav je neizrazit vrh v glavnem greben...  
234  Visoki Rokav je najvišja gora v Sloveniji na k...  
235  Jalovec je z višino 2645 m, šesta najvišja gor...  
..                                                 ...  
535  Izvir Glijuna je močan kraški izvir, ki se nah...  
536  Mala Boka je kraška jama, ki se nahaja ob vzno...  
537  Kozlov rob ali Grad kot mu pravijo domačini, j...  
538  Soča je na tem mestu izdolbla korita dolga okr...  
539  Tolminska korita so dolga 200 m, široka pa od ...  

[309 rows x 9 columns]

In [24]:
st_julijcev = julijci['ime'].count()
st_julijcev

309

Vrhov v julijskih alpah bi moralo biti 565 (toliko je shranjenih strani), v tabeli jih imamo le 309. Preden iščemo napako pregeljmo še ostale vrhove.

In [26]:
kamniske = hribi[hribi.gorovje == 'Kamniško Savinjske Alpe']
st_kamniskih = kamniske['ime'].count()
st_kamniskih

154

Vrhov v Kamniško Savinjskih Alpah bi morali imeti 281, v tabeli jih je le 154.

In [28]:
karavanke = hribi[hribi.gorovje == 'Karavanke']
st_karavank = karavanke['ime'].count()
st_karavank

145

Vrhov v Karavankah bi moralo biti 362 in ne zgolj 145.

In [33]:
gns = hribi[hribi.gorovje == 'Goriško, Notranjsko in Snežniško hribovje']
st_gns = gns['ime'].count()
st_gns

231

Vrhov v Goriškem, Notranjskem in Snežniškem hribovju bi moralo biti 292, imamo jih le 231.

In [34]:
pohorje = hribi[hribi.gorovje == 'Pohorje, Dravinjske gorice in Haloze']
st_pohorje = pohorje['ime'].count()
st_pohorje

55

Namesto 95 imamo le 55 vrhov na Pohorju, Dravinjskih goricah in Halozah :(.

In [35]:
pl = hribi[hribi.gorovje == 'Polhograjsko hribovje in Ljubljana']
st_pl = pl['ime'].count()
st_pl

69

Namesto 88 imamo v Polhograjskem hribovju in v Ljubljani 69 vrhov.

In [36]:
pd = hribi[hribi.gorovje == 'Posavsko hribovje in Dolenjska']
st_pd = pd['ime'].count()
st_pd

181

Namesto 230 jih je v Posavskem hribovju in na Dolenjskem le 181.

In [37]:
prekmurje = hribi[hribi.gorovje == 'Prekmurje']
st_prekmurje = prekmurje['ime'].count()
st_prekmurje

37

Namesto 54 jih je v Prekmurju 37.

In [38]:
scj = hribi[hribi.gorovje == 'Škofjeloško, Cerkljansko hribovje in Jelovica']
st_scj = scj['ime'].count()
st_scj

91

Namesto 143 jih je zgolj 91 v Škofjeloškem, Cerkljanskem hribovju in v Jelovici.

In [39]:
skks = hribi[hribi.gorovje == 'Strojna, Košenjak, Kozjak in Slovenske gorice']
st_skks = skks['ime'].count()
st_skks

50

In nazadnje jih je namesto 74 le 50 v Strojni, Košenjaku, Kozjaku in Slovenskih goricah.